packages/ui/src/pages/Home/MultisigView.tsx

```diff
@@ -4,52 +4,55 @@ import MultisigActionMenu from './MultisigActionMenu'
 import { styled } from '@mui/material/styles'
 import { Box, Chip } from '@mui/material'
 import { useMultiProxy } from '../../contexts/MultiProxyContext'
-import Balance from '../../components/library/Balance'
-import { renderMultisigHeading } from '../multisigHelpers'
 import MultisigAccordion from './MultisigAccordion'
+import { Balance } from '../../components/library'
 
 const MultisigView = () => {
-  const { selectedMultiProxy, selectedHasProxy, selectedIsWatched } = useMultiProxy()
+  const { selectedMultiProxy, selectedHasProxy } = useMultiProxy()
 
   return (
     <MultisigViewWrapperStyled>
       <HeaderStyled>
-        {selectedMultiProxy && (
-          <h3>
-            {selectedHasProxy
-              ? 'Controlled by'
-              : renderMultisigHeading(selectedMultiProxy.multisigs.length > 1)}
-          </h3>
-        )}
-        {!selectedHasProxy && (
+        {selectedMultiProxy && <h3>{selectedHasProxy ? 'Controlled by' : 'Account Details'}</h3>}
+        {selectedHasProxy && (
           // only show the buttons if we have a solo multisig
           // and only show the send button if we are part of the multisig
           <BoxStyled>
-            <MultisigActionMenu withSendButton={!selectedIsWatched} />
+            <MultisigActionMenu
+              menuButtonBorder={'none'}
+              withNewTransactionButton={false}
+            />
           </BoxStyled>
         )}
       </HeaderStyled>
       <MultisigList>
         {selectedMultiProxy &&
           selectedMultiProxy.multisigs.map((multisig) => {
             return (
-              <MultisigWrapperStyled key={multisig.address}>
-                <AccountDisplayWrapperStyled>
-                  <AccountDisplay
-                    address={multisig.address || ''}
-                    badge={AccountBadge.MULTI}
-                    withBalance={false}
-                  />
-                </AccountDisplayWrapperStyled>
+              <MultisigWrapperStyled
+                isPureMultisig={selectedHasProxy}
```

Comment:
can we keep the same naming?
```suggestion
                selectedHasProxy={selectedHasProxy}
```

Reply:
Actually, I am considering having a descriptive name for the conditions we put for multisig or pure multisig as the name selectedHasProxy requires knowledge of the project to understand from where the word is coming.

Every time I do the condition I should mentally remember:
it has a proxy => which means it's pure multisig => because only pure multisig can have proxy

Instead, we could return "type" from Context

export interface IMultisigContext {
   multisigType: "pure" | "solo"
}

For current PR I will use selectedHasProxy

Reply:
I called the overall object MultiProxy, because it's a tree with "pure > multisigs []" where the pure could be empty. We could for sure add some more derived state, but this may make this tree a little bloated maybe? So far we mostly needed to know whether it's a pure (with several multisigs) or if it's a single Multisig, and this is in the `selectedHasPure`. Similarly we have `selectedIsWatched`. The main thing I see is that we'd need eventually a `selectedDisplayAddress`, which is some code that's duplicated with `selectedIsWatched ? selected.pure : selected.multisigs[0].address`. 

But yes this should 100% be for another PR.